### PR TITLE
chore: added `tombstone-collection` command to `cxg_admin.py` (#1466)

### PR DESCRIPTION
**Pull request by metakuni**
### Reviewers
**Functional:** 
@MDunitz @ebezzi 

**Readability:** 
@blrnw3 

---

## Changes
- added: `tombstone-collection` command

### CODE REVIEW

File: scripts/cxg_admin.py [hunk 1]
```diff
@@ -76,6 +76,51 @@ def delete_dataset(ctx, uuid):
             exit(0)
 
 
+@cli.command()
+@click.argument("uuid")
+@click.pass_context
+def tombstone_collection(ctx, uuid):
```
Comment (blrnw3) [inline scripts/cxg_admin.py:82]: nit: type hints on the parameters

File: scripts/cxg_admin.py [hunk 2]
```diff
@@ -76,6 +76,51 @@ def delete_dataset(ctx, uuid):
...
+    """
+    Tombstones the collection specified by UUID.
+
+    You must first SSH into the target deployment using `make db/tunnel` before running.
```
Comment (blrnw3) [inline scripts/cxg_admin.py:86]: I know this is used a lot but it always confused me. For me, `SSH`  when used as a verb means to literally log in to the remote machine, not make a tunnel :D

File: scripts/cxg_admin.py [hunk 3]
```diff
@@ -76,6 +76,51 @@ def delete_dataset(ctx, uuid):
...
+    """
+
+    with db_session_manager() as session:
+        collection = Collection.get_collection(session, uuid, include_tombstones=True)
+
+        if not collection:
+            click.echo(f"Collection:{uuid} not found!")
+            exit(0)
+        else:
+            print(
```
Comment (blrnw3) [inline scripts/cxg_admin.py:100]: nit: `click.echo` for consistency. Pretty sure they do the same thing so you could make everything a print if you like :D

**Review by blrnw3 — APPROVED** [review]
Readability comments inline. Functionality looks right to me too 🤷

File: scripts/cxg_admin.py (hunk 2 above)
Comment (metakuni) [reply scripts/cxg_admin.py]: Fixed

File: scripts/cxg_admin.py (hunk 3 above)
Comment (metakuni) [reply scripts/cxg_admin.py]: Ah right, fixed.

File: Dockerfile.processing_base [hunk 4]
```diff
@@ -20,5 +20,8 @@ RUN apt-get update && apt-get -yq install \
 RUN R -e "install.packages('BiocManager')"
 RUN R -e "BiocManager::install('SingleCellExperiment')"
 RUN R -e "install.packages(c('devtools', 'igraph'));"
+# note: Loom format has been deprecated from cellxgene but cellgeni/sceasy has a dependency on LoomExperiment
+#       See https://github.com/cellgeni/sceasy#installation
+RUN R -e "BiocManager::install('LoomExperiment')"
```
Comment (metakuni) [inline Dockerfile.processing_base:25]: This has nothing to do with tombstoning of collections, but this PR's build was broken by removing this dependency in #1465 , so re-adding it here.

Comment (metakuni) [reply Dockerfile.processing_base:24]: ... and now, the bioconductor.org site appears to be down causing dependency installation to fail: 🤦 
```
...
#8 490.1   cannot open URL 'https://bioconductor.org/packages/3.12/bioc/src/contrib/LoomExperiment_1.8.0.tar.gz': HTTP status was '504 Gateway Timeout'
...
```

**Review by MDunitz — APPROVED** [review]
lgtm

**Pull request merged**